### PR TITLE
create scenario guide template file

### LINKED ISSUE #46122: Create a scenario guide template
##### SUMMARY

Since scenario guides are a popular documentation addition from third-party and partners, create a reusable rst template they can use to start their scenario guides to ensure a common flow and priority information is included.

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME

docs.ansible.com
##### ANSIBLE VERSION

```paste below
2.8
```

##### CONFIGURATION

```paste below

```

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Add an rst  file with outline for a scenario guide.  

Fixes #46122

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs.ansible.com
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by samccann** [comment]
@Akasurde - this PR moves the vmware scenario template into the scenario_guide folder and makes a few adjustments (as per the issue) to make it generic. Please take a look and let me know if you are okay with this move.  Thanks!

**Review by Akasurde — APPROVED** [review]
LGTM

File: docs/docsite/rst/scenario_guides/scenario_template.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+What the user should expect to see.
+
+
+Troubleshooting
+---------------
+
+What to look for if it breaks.
+
+
+Conclusion and Where To Go Next
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/scenario_template.rst]:
```suggestion
Conclusion and where to go next
```

File: docs/docsite/rst/scenario_guides/scenario_template.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+=============
+
+Describe the requirements and assumptions for this scenario. This should include applicable subsections for hardware, software, and any other caveats to using the scenarios in this guide.
+
+Credentials and authenticating
+==============================
+
+Describe credential requirements and how to authenticate to this platform.
+
+Using a dynamic inventory
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/scenario_template.rst]:
```suggestion
Using dynamic inventory
```

**Review by acozine — COMMENTED** [review]
This is an improvement over what we have now, for sure. As we keep talking about what we want platform documentation to look like, let's keep updating this work.

**Pull request merged**